CONTRIBUTING.md

```diff
@@ -236,9 +236,23 @@ To fix formatting issues in code use this command:-
 
 We are using the package `Husky` to run a pre-commit hook which automatically runs these checks each time you make a commit and also fixes some of the issues. This way you don't have to run them manually each time.
```

Comment:
Change the details to just mention that we're running hooks that run according to different git workflows.